payloads/v10/channel.ts

```diff
@@ -1990,6 +2080,18 @@ export type APIMessageActionRowComponent = APIButtonComponent | APISelectMenuCom
 // Modal components
 export type APIModalActionRowComponent = APITextInputComponent;
 
+export type APISectionAccessoryComponent = APIButtonComponent | APIThumbnailComponent;
+
+export type APIContainerInnerComponent =
+	| APIActionRowComponent<APIMessageActionRowComponent>
+	| APIFileComponent
+	| APIMediaGalleryComponent
+	| APISectionComponent
+	| APISeparatorComponent
+	| APITextDisplayComponent;
+
+export type APIMessageTopLevelComponent = APIContainerComponent | APIContainerInnerComponent;
```

Comment:
Similar to my previous comment, I'd recommend listing the components manually. Seems easy to end up in a bad situation if we added a component that can only go in a Container and due to this union your typing says it can go anywhere at the top-level too.